admin-cli/build.gradle

```
@@ -7,6 +7,8 @@ dependencies {
     implementation project(':client')
     implementation project(':common')
     implementation 'com.networknt:json-schema-validator:1.0.73'
+    implementation 'org.springframework.boot:spring-boot'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```

Looking a the classes this subproject needs I think you can replace both lines with
```suggestion
    implementation 'org.springframework.boot:spring-boot-autoconfigure'
```